mountinfo/mountinfo_bsd.go

```diff
@@ -1,43 +1,38 @@
-//go:build (freebsd && cgo) || (openbsd && cgo) || (darwin && cgo)
-// +build freebsd,cgo openbsd,cgo darwin,cgo
+//go:build freebsd || openbsd || darwin
+// +build freebsd openbsd darwin
 
 package mountinfo
 
-/*
-#include <sys/param.h>
-#include <sys/ucred.h>
-#include <sys/mount.h>
-*/
-import "C"
+import "syscall"
 
-import (
-	"fmt"
-	"reflect"
-	"unsafe"
-)
+func getInfo(is []int8) string {
```

Comment:
This function name isn't descriptive. 
`func stringFromInt8Slice([]int8) string` might be better.

Reply:
I renamed it to int8SliceToString.